utils/git/Invoke-MergeTogether.mocks.psm1

```diff
@@ -0,0 +1,85 @@
+Import-Module -Scope Local "$PSScriptRoot/../core.psm1"
+Import-Module -Scope Local "$PSScriptRoot/../testing.psm1"
+Import-Module -Scope Local "$PSScriptRoot/../query-state.psm1"
+Import-Module -Scope Local "$PSScriptRoot/../query-state.mocks.psm1"
+Import-Module -Scope Local "$PSScriptRoot/Invoke-MergeTogether.psm1"
+
+function Invoke-MockGitModuleStartsWith([string] $gitCli, [object] $MockWith) {
+    $result = New-VerifiableMock `
+        -ModuleName 'Invoke-MergeTogether' `
+        -CommandName git `
+        -ParameterFilter $([scriptblock]::Create("(`$args -join ' ').StartsWith('$gitCli')"))
```

Comment:
Is there a reason this needs to be created from a string? I don't see threading?

Reply:
When doing a CLI mock, you just get an `$args` variable. This allows us to make a mock that _looks_ pretty close to the CLI (minus some quotes, etc.) and still mock the `git` command.